docs/source/release/v6.2.0/muon.rst

```diff
@@ -11,54 +11,51 @@ Frequency Domain Analysis
+.. image::  ../../images/maxent_dual_plot.png
+   :align: center
+   :height: 800px
+
+Improvements
+############
+
+- Frequency Domain Analysis can now use groups in :ref:`MuonMaxent <algm-MuonMaxent>` calculations.
+- It is easier to select data for Maxent calculations.
+
+Bugfixes
+########
 
 - In frequency domain analysis the phasetables calculated from :ref:`MuonMaxent <algm-MuonMaxent>` can be used for
   :ref:`PhaseQuad <algm-PhaseQuad>` calculations on the phase tab.
 
-Muon Analysis 2
----------------
+Muon Analysis
+-------------
 
 New Features
 ############
 
-- Users can now copy sequential fitting parameters to all other runs. To do this, click the checkbox to turn this feature
-  on and change a parmeter. The change should be copied to the other cells in the same column. Turn the feature off to make
-  changes to single rows.
-- The :ref:`Model Fitting Tab <muon_model_fitting_tab-ref>` allows you to perform fits across the sample logs and fit parameters stored in your results table.
+- Users can now copy sequential fitting parameters to all other runs using the ``Copy fit parameters to all`` checkbox.
+- **The** :ref:`Model Fitting Tab <muon_model_fitting_tab-ref>` **allows you to perform fits across the sample logs and fit parameters stored in your results table.**
 
 .. image::  ../../images/muon_model_fitting_tab.PNG
    :align: center
    :height: 800px
 
-Improvements
-############
 
-- When running the Dynamic Kubo Toyabe fit function you should now be able to see the BinWidth to 3 decimal places.
-- Can now select the normalisation (``analysis_asymmetry_norm``) and group (``analysis_group``) in the results table tab.
-
-Bug Fixes
-#########
-- A bug has been fixed in the BinWidth for the Dynamic Kobu Toyabe Fitting Function which caused a crash and did not provide
-  any information about why the value was invalid. Will now revert to last viable BinWidth used and explain why.
-- The autoscale option when `All` is selected will now show the largest and smallest y value from the all of the plots.
-
-Muon Analysis 2 and Frequency Domain Analysis
----------------------------------------------
+Muon Analysis and Frequency Domain Analysis
+-------------------------------------------
 
 New Features
 ############
 
-- It is now possible to Exclude a range from a fit range when doing a fit on the Fitting tab.
-- Added a 'Covariance Matrix' button to the Fitting tab that can be used to open and inspect the normalised covariance parameters of a fit.
-- Can now plot the raw count data in the GUI.
-- It is now possible to perform an Automatic or Manual background correction in the new :ref:`Corrections Tab <muon_corrections_tab-ref>`.
+- It is now possible to exclude a single range from a fit range when doing a fit on the :ref:`Fitting Tab <muon_fitting_tab-ref>`.
+- Added a 'Covariance Matrix' button to the :ref:`Fitting Tab <muon_fitting_tab-ref>` that can be used to open and inspect the normalised covariance parameters of a fit.
```

Comment:
Can you change the single quotes to be \`\`Covariance Matrix\`\` instead. That is my fault for using the wrong thing previously